### ISSUE TITLE
Unnecessary files

### ISSUE THREAD

**Issue by zhurlik** [valentine-ezugu/onlinestore#1]
Could you delete IDEA files that can be generated by maven

- .idea
- *.iml

**Comment by valentine-ezugu** [comment]
handling now thank you

**Comment by zhurlik** [comment]
I still see [.idea](https://github.com/valentine-ezugu/onlinestore/tree/master/.idea)

**Comment by valentine-ezugu** [comment]
Yes I use gitIgnore to remove doesn't remove I will check how to do this today and remove

**Comment by valentine-ezugu** [comment]
Thank you
